autodoc/admin-api/data/admin-api.lua

```diff
@@ -669,6 +669,37 @@ return {
               means Kong does not currently have a valid configuration loaded.
           ]],
         },
+      },
+      ["/status/ready"] = {
+        GET = {
+          title = [[Check node readiness]],
+          endpoint = [[<div class="endpoint get">/status/ready</div>]],
```

Comment:
Should this document go in here? Isn't this code specific to Admin API?

Reply:
> Should this document go in here? Isn't this code specific to Admin API?

cause we don't have a Status API specific doc yet

Reply:
We are documenting incorrectly. We cannot document an endpoint on Admin API which doesn't exist, this only confuses the user as well as maintainers.

Please remove this document and add it manually to docs.konghq.com.

@mheap Any suggestions?

Reply:
I'll tag in @Kong/docs-maintainers for where to add it.

Is this a brand new API, or a part of something that already exists?

Reply:
> I'll tag in @Kong/docs-maintainers for where to add it.
> 
> Is this a brand new API, or a part of something that already exists?

It is a new endpoint

Reply:
@hbagdi stated that it shouldn’t be part of the admin API docs. I don’t have enough context to know where it should go

Any pointers @hbagdi?

Reply:
Status API has been around for a long time but it is under-documented.
Status API is configured via the 'status_listen' kong.conf property.
This PR adds a new endpoint to the Status API (and not Admin API).

We need to figure out where (if) to document this endpoint.
The fact that this endpoint doesn't exist on Admin API should be sufficient to not add this endpoint to Admin API documentation.

Reply:
@Kong/docs-maintainers FYI - looks like we have another API to document that's currently missing

Reply:
Hit enter too soon... 

@hbagdi we should document if it's something we offer to customers. Let's start with this endpoint and expand to others as we gain an understanding of what they do and how they're used